sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -138,16 +139,35 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers, ContactService contac
           selectVaccinationStatusForThisDisease(editedContact.getVaccinationStatusForThisDisease());
           selectImmunosuppressiveTherapy(editedContact.getImmunosuppressiveTherapy());
           selectActiveInCare(editedContact.getActiveInCare());
-          clickCancelFollowUpButton();
-          // TODO enable it back once 6803 is fixed
-          // selectOverwriteFollowUp(editedContact.getOverwriteFollowUp());
-          // fillDateOfFollowUpUntil(editedContact.getDateOfFollowUpUntil());
-          fillFollowUpStatusComment(editedContact.getFollowUpStatusComment());
-          selectResponsibleContactOfficer(editedContact.getResponsibleContactOfficer());
-          fillGeneralComment(editedContact.getGeneralComment());
-          webDriverHelpers.clickOnWebElementBySelector(SAVE_EDIT_BUTTON);
-          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(SAVE_EDIT_BUTTON);
+            clickCancelFollowUpButton();
+            // TODO enable it back once 6803 is fixed
+            // selectOverwriteFollowUp(editedContact.getOverwriteFollowUp());
+            // fillDateOfFollowUpUntil(editedContact.getDateOfFollowUpUntil());
+            fillFollowUpStatusComment(editedContact.getFollowUpStatusComment());
+            selectResponsibleContactOfficer(editedContact.getResponsibleContactOfficer());
+            fillGeneralComment(editedContact.getGeneralComment());
+            webDriverHelpers.clickOnWebElementBySelector(SAVE_EDIT_BUTTON);
+            webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(SAVE_EDIT_BUTTON);
         });
+      When(
+              "I click on Confirm Contact radio button Contact Person tab",
+              () -> webDriverHelpers.clickOnWebElementBySelector(CASE_CONFIRMED_RADIO_BUTTON)
+      );
+      When(
+              "^I click SAVE button$",
```

Comment:
specify also the location of the save button, to avoid conflicts with other similar namings (which will be refactored)